Task2calculatorIF/src/Task2calculatorIF.java

```diff
@@ -0,0 +1,39 @@
+
+    import java.util.Scanner;
+
+    public class Task2calculatorIF {
```

Comment:
If you go with CamelCase approach in naming Class, then First word capital, other lowercase (e.g.Task2CalculatorIf)